content/v1.14/concepts/composition-functions.md

```diff
@@ -452,10 +452,8 @@ $ crossplane xpkg push -f package/*.xpkg crossplane-contrib/function-example:v0.
 ```
 
 {{<hint "tip">}}
-Crossplane has
-[language specific guides]({{<ref "../../knowledge-base/guides">}}) to writing
-a composition function. Refer to the guide for your preferred language for a
-more detailed guide to writing a function.
+Crossplane has a 
+[guide to writing a composition function in Go]({{<ref "../../knowledge-base/guides">}}). 
```

Comment:
looks like this is still pointing to the configuration guides high level parent topic, not directly to the golang guide 🕵️

Reply:
🤦‍♂️
fixed.